### PR TITLE
Popularity table

### PR DESCRIPTION
**Pull request by LucvandenBrand**
Changed the _popularity VS as-shown-to-user_ table into something that's hopefully a bit less confusing, and has a much higher resolution because it is made in latex itself:
image

### CODE REVIEW

**Comment by mircealungu** [comment]
Cool idea! 
The only problem, if I scale to 0.5 (so the font is comparable with the surrounding text's) the two columns of text come too close together, and the arrows are overlapping the text.

File: results.tex [hunk 1]
```diff
@@ -29,7 +29,40 @@ \section{Do The Readers Find Personally Interesting Articles?}
...
+  \begin{tikzpicture}[scale=1]
+    % Columns.
+    \node at (0  , 0) {\bf Presented};
+    \node at (2.5, 0) {\bf VS};
+    \node at (5  , 0) {\bf Popularity};
+    
+    % As presented.
+    \node at (0,-1) {1Jour1Actu};
+    \node at (0,-2) {L'Equipe};
+    \node at (0,-3) {La Bogoth\`{e}que};
```
Comment (mircealungu) [inline results.tex]: B_l_ogotheque

**Comment by LucvandenBrand** [comment]
Thanks for the feedback, both issues should have been fixed now :)

**Comment by mircealungu** [comment]
goed. made one more commit to this branch, and will merge now.

**Pull request merged**